projects/plugins/boost/app/features/optimizations/Optimizations.php

```diff
@@ -103,7 +103,8 @@ public function setup() {
 				continue;
 			}
 
-			$optimization->feature->setup();
+			add_action( $optimization->feature->setup_trigger(), array( $optimization->feature, 'setup' ) );
```

Comment:
There is an `Automattic\Jetpack_Boost\Lib\Setup` which has this implemented this `add_action`. The advantage of using that over hooking directly is that using the `Setup` class allows you to `remove_action` on any of the instances' methods or to get access to the exact instance which was setup for any other reason.

Reply:
That actually makes sense! Initially when debugging, I wasn't sure why setuppable features aren't present in the instances list.

I've fixed this with 380724d9219c364e29b5d8f3f6c5d6ea13da11f2.